keepassxc-browser/popups/popup_functions.js

```diff
@@ -48,7 +48,7 @@ async function initColorTheme() {
 }
 
 async function getLoginData() {
-    const tab = getCurrentTab();
+    const tab = await getCurrentTab();
     if (!tab) {
         return [];
     }
```

Comment:
I don't know if this mistake had any adverse impact. It would just mean that the `if (!tab) {` check never did anything, as `tab` is always a `Promise` here.

If the source code was converted to TypeScript then these mistakes could be detected with the proper eslint packages. Has that ever been considered?

Reply:
> f the source code was converted to TypeScript then these mistakes could be detected with the proper eslint packages. Has that ever been considered?

At one point, yes. There's just two small problems with this:
1) I have never used TypeScript.
2) I like to keep the code in vanilla JS so it's easy to debug on-the-fly (by anyone) and the code size stays small.